Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1202,48 +1202,75 @@ namespace Babylon
             XRRay(const Napi::CallbackInfo& info)
                 : Napi::ObjectWrap<XRRay>{info}
             {
-                bool originSet = false;
-                bool directionSet = false;
-                bool matrixSet = false;
-                if (info[0].IsObject())
+                auto argLength = info.Length();
```

Comment:
nit:
```suggestion
                auto argLength{info.Length()};
```

Reply:
Some places below as well.

Reply:
What is the general guideline for using "{}" instead of "=" in Babylon code?

Reply:
This is not guidelines for Babylon code but C++ in general: 
https://stackoverflow.com/questions/18222926/why-is-list-initialization-using-curly-braces-better-than-the-alternatives